tests/unit_tests/subtensor/interface/test_keypair.py

```diff
@@ -206,9 +208,9 @@ def test_hdkd_path_gt_32_bytes(self):
     def test_hdkd_unsupported_password(self):
         self.assertRaises(NotImplementedError, Keypair.create_from_uri, DEV_PHRASE + '///test')
 
-    def test_reconstruct_path_fail(self):
-        self.assertRaises(ValueError, extract_derive_path, 'no_slashes')
-        self.assertRaises(ValueError, extract_derive_path, '//')
+    # def test_reconstruct_path_fail(self):
+    #     self.assertRaises(ValueError, extract_derive_path, 'no_slashes')
+    #     self.assertRaises(ValueError, extract_derive_path, '//')
```

Comment:
Also removing this, it's not really required since the new substrate takes care of these paths in the backend, and `extract_derive_path` is no longer exposed this way.